lib/archethic/contracts/interpreter/version1/condition_interpreter.ex

```diff
@@ -179,22 +173,20 @@ defmodule Archethic.Contracts.Interpreter.Version1.ConditionInterpreter do
 
   # Override Contract.get_calls()
   defp postwalk(
-         _subject = [global_variable, _],
-         _node =
+         _subject = [_global_variable, _],
+         node =
            {{:., _meta, [{:__aliases__, _, [atom: "Contract"]}, {:atom, "get_calls"}]}, _, []},
-         acc
+         _acc
        ) do
-    new_node =
-      quote do
-        Archethic.Contracts.Interpreter.Version1.Library.Contract.get_calls(
-          get_in(
-            Process.get(:scope),
-            [unquote(global_variable), "address"]
-          )
-        )
-      end
-
-    {new_node, acc}
+    # new_node =
```

Comment:
I disabled `Contract.get_calls()` from the `condition` blocks for now, because it'd require a quorum and not a read from DB. (because the `condition inherit` validator is not the worker).